sale_blanket_order/migrations/14.0.1.2.2/pre-migrate.py

```diff
@@ -16,4 +16,3 @@ def migrate(env, version):
         WHERE sbo.state = 'expired'
         """,
     )
-    env.cr.commit()
```

Comment:
**issue**: This is a breaking change, it definitely deserves a major version --> `14.0.2.0.0`. Also, when you add a migration script, you need to also change the version in the `__manifest__.py` manually to make sure they are the same.